qiskit/wrapper/_wrapper.py

```diff
@@ -202,3 +206,56 @@ def execute(circuits, backend,
     q_job = QuantumJob(qobj, backend=backend, preformatted=True, resources={
         'max_credits': qobj['config']['max_credits']})
     return backend.run(q_job)
+
+
+# Functions for importing qasm
+
+
+def load_qasm_string(qasm_string, name=None,
+                     basis_gates="id,u0,u1,u2,u3,x,y,z,h,s,sdg,t,tdg,rx,ry,rz,"
+                                 "cx,cy,cz,ch,crz,cu1,cu3,swap,ccx,cswap"):
+    """Construct a quantum circuit from a qasm representation (string).
+
+    Args:
+        qasm_string (str): a string of qasm, or a filename containing qasm.
+        basis_gates (str): basis gates for the quantum circuit.
+        name (str or None): the name of the quantum circuit after loading qasm
+            text into it. If no name given, assign automatically.
+    Returns:
+        QuantumCircuit: circuit constructed from qasm.
+    """
+    node_circuit = Qasm(data=qasm_string).parse()
+    unrolled_circuit = Unroller(node_circuit, CircuitBackend(basis_gates.split(",")))
+    circuit_unrolled = unrolled_circuit.execute()
+    if name:
+        circuit_unrolled.name = name
+    return circuit_unrolled
+
+
+def load_qasm_file(qasm_file, name=None,
+                   basis_gates="id,u0,u1,u2,u3,x,y,z,h,s,sdg,t,tdg,rx,ry,rz,"
+                               "cx,cy,cz,ch,crz,cu1,cu3,swap,ccx,cswap"):
+    """Construct a quantum circuit from a qasm representation (file).
+
+    Args:
+        qasm_file (str): a string for the filename including its location.
+        name (str or None): the name of the quantum circuit after
+            loading qasm text into it. If no name is give the name is of
+            the text file.
+        basis_gates (str): basis gates for the quantum circuit.
+    Returns:
+        str: Adds a quantum circuit with the gates given in the qasm file to the
```

Comment:
Can you revise this line? It seems the `QuantumProgram` function was returning a `str`, but now the function returns a `QuantumCircuit` (which sounds good!).